test/jdk/java/awt/TrayIcon/ShowAfterDisposeTest/ShowAfterDisposeTest.java

```diff
@@ -22,68 +22,77 @@
+    public static void main(String[] args) throws Exception {
+        ShowAfterDisposeTest test = new ShowAfterDisposeTest();
+        test.startTest();
+    }
 
-        String[] instructions;
+    public void startTest() throws Exception {
         traySupported = SystemTray.isSupported();
-        if (traySupported)
-        {
-            String[] s =
-            {
-                "1) When the test starts an icon is added to the SystemTray area.",
-                "2a) If you use Apple OS X,",
-                "    right click on this icon (it's important to click before the tooltip is shown).",
-                "    The icon should disappear.",
-                "2b) If you use other os (Windows, Linux, Solaris),",
-                "    double click on this icon (it's important to click before the tooltip is shown).",
-                "    The icon should disappear.",
-                "3) If the bug is reproducible then the test will fail without assistance.",
-                "4) Just press the 'pass' button."
-            };
-            instructions = s;
-        }
-        else
-        {
-            String[] s =
-            {
-              "The test cannot be run because SystemTray is not supported.",
-              "Simply press PASS button."
-            };
-            instructions = s;
+        if (!traySupported) {
+            System.out.println("The test cannot be run because SystemTray is not supported.");
+            return;
         }
-        Sysout.createDialogWithInstructions(instructions);
-    }
 
-    public void start ()
-    {
-        setSize (200,200);
-        setVisible(true);
-        validate();
+        String instructions =
+            "1) When the test starts an icon is added to the SystemTray area.\n" +
+            "2a) If you use Apple OS X,\n" +
+            "    right click on this icon (it's important to click before the tooltip is shown).\n" +
+            "    The icon should disappear.\n" +
+            "2b) If you use other os (Windows, Linux, Solaris),\n" +
+            "    double click on this icon (it's important to click before the tooltip is shown).\n" +
+            "    The icon should disappear.\n" +
+            "3) If the bug is reproducible then the test will fail without assistance.\n" +
+            "4) Just press the 'pass' button.";
+
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
+                .title("Test Instructions Frame")
+                .instructions(instructions)
+                .testTimeOut(10)
+                .rows(10)
+                .columns(45)
+                .build();
+        showFrameAndIcon();
+        passFailJFrame.awaitAndCheck();
+    }
 
-        if (!traySupported)
-        {
+    public void showFrameAndIcon() throws Exception {
+        if (!traySupported) {
             return;
         }
 
+        SwingUtilities.invokeAndWait(() -> {
+            frame = new JFrame();
+            frame.setLayout (new BorderLayout());
```

Comment:
```suggestion
            frame.setLayout(new BorderLayout());
```
No need of space here and below as well.